### ISSUE TITLE
Only Visualization of Loss is demonstrated in Tensorboard Scalars Documentation

### ISSUE THREAD

**Issue by worldpeaceaspirer** [tensorflow/tensorboard#4901]
## Issue description:
This is an issue to improve [documentation of Tensorboard Scalars](https://www.tensorflow.org/tensorboard/scalars_and_keras). In [this Tutorial](https://www.tensorflow.org/tensorboard/scalars_and_keras), only visualization of **`Loss`** is demonstrated but usually, we visualize **`Metrics`** as well (like `Accuracy`).

So, there can be a mention on how to visualize Metrics.

**Comment by wchargin** [comment]
Hi @worldpeaceaspirer! The TensorBoard Keras callback used in those docs
will automatically log all the metrics that you’re tracking. So, if you
want to track accuracy, just add `metrics=["accuracy"]` to your model
creation code:

```python
model.compile(
    loss='mse',
    optimizer=keras.optimizers.SGD(lr=0.2),
    metrics=['accuracy'],
)
```

…and you’ll see `epoch_accuracy` appear in TensorBoard.

Does this answer your question?

**Comment by worldpeaceaspirer** [comment]
@wchargin,
Thank you for your response and sorry for the delayed response. My point is, since the purpose of documentation is to guide the community, it will be better to cover all the Cases like **`Metrics`** and **`Loss`**, instead of just **`Loss`**, so that the documentation would be complete.
